rails_app/db/migrate/20230420085950_change_relations_use_pk.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+class ChangeRelationsUsePk < ActiveRecord::Migration[7.0]
+  def change
+    change_table :currency_followings do |t|
+
+      t.remove_index([:follower_email, :followed_currency_code])
+
+      t.remove_foreign_key column: :follower_email if t.foreign_key_exists? column: :follower_email
+
+      t.remove_foreign_key column: :followed_currency_code if t.foreign_key_exists? column: :followed_currency_code
```

Comment:
What's the reason to need those conditionals? I'd say at the time of the migration you can assume the FK exists. Otherwise we wouldn't be creating a migration to remove it.

Reply:
:+1: